lib/src/client.dart

```diff
@@ -132,7 +132,7 @@ abstract class Nyxx {
 }
 
 /// A client that can make requests to the HTTP API and is authenticated with a bot token.
-class NyxxRest with ManagerMixin implements Nyxx {
+class NyxxRest with ManagerMixin, EventRestMixin implements Nyxx {
```

Comment:
I'd rather not all `NyxxRest` clients have the `onInteractionCreate` stream as most HTTP only clients will not be receiving the events.

If the events are only emitted if a plugin is added to the client, then the stream should be on the plugin itself.

Reply:
yeah,  stream could be added to plugin but then implementation for nyxx commands would be hard or impossible. We have to have some kind of support in main Nyxx interfaces. Also most client might not receive events but it's supported feature - I dont see problem with exposing possible feature even if rarely it would be used.